src/index.jsx

```diff
@@ -3,6 +3,14 @@ import ReactDOM from "react-dom/client";
 import { Provider } from "react-redux";
 import store from "./store";
 import App from "./components/App";
+import copy from "./common/data/copy.json";
+
+// XXX: Hack to make migration from "copy.json" and
+// adding missing translation strings smoother.
+Object.assign(copy, {
+  ru: { ...copy["en-US"], ...copy["uk"], ...copy["ru"] },
+  uk: { ...copy["en-US"], ...copy["ru"], ...copy["uk"] },
+});
```

Comment:
There are a few obsolete strings in `copy.json` and many more not there, but in `config.js` or other places.
I think the best way to move forward is not changing it, but:

- create new files (separate for each language to aid with diffs)
- put them together in one `translations` object (merge `uk <-> ru -> en` for fallbacks)
- go over the codebase updating each usage of `copy.json` to use `translations` and adding necessary translation strings
- delete `copy.json` when done

Reply:
I'm comfortable with that refactoring 👍

Reply:
Also I realize that `copy.json` (and other language files under `src/common/`) shouldn't contain translations specific to our instance of timemap, those should go into `config.js`. Changes in `copy.json` should be restricted to generic timemap strings to be backportable. I'll keep this in mind.